File: app/src/main/java/com/bogdanorzea/popularmovies/model/response/MoviesResponse.java
package com.bogdanorzea.popularmovies.model.response;

import com.bogdanorzea.popularmovies.model.object.TruncatedMovie;
import com.squareup.moshi.Json;

import java.util.List;

public class MoviesResponse {

    @Json(name = "page")
    public int page;
    @Json(name = "total_results")
    public int totalResults;
    @Json(name = "total_pages")
    public int totalPages;
    @Json(name = "results")
    public List<TruncatedMovie> results = null;

}

File: app/src/main/java/com/bogdanorzea/popularmovies/model/response/MoviesResponse.kt
package com.bogdanorzea.popularmovies.model.response

import com.bogdanorzea.popularmovies.model.`object`.Movie
import com.squareup.moshi.Json

data class MoviesResponse(@Json(name = "page") var page: Int = 0,
                          @Json(name = "total_results") var totalResults: Int = 0,
                          @Json(name = "total_pages") var totalPages: Int = 0,
                          @Json(name = "results") var results: List<Movie>? = null)
